shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -717,6 +770,8 @@ void Initialize(v8::Local<v8::Object> exports,
   v8::Isolate* isolate = context->GetIsolate();
   gin_helper::Dictionary dict(isolate, exports);
   dict.SetMethod("exposeAPIInMainWorld", &electron::api::ExposeAPIInMainWorld);
+  dict.SetMethod("exposeAPIInIsolatedWorld",
+                 &electron::api::ExposeAPIInIsolatedWorld);
```

Comment:
Since both `exposeAPIInMainWorld` and `exposeAPIInIsolatedWorld` have almost the same implementation, I'd be in favor of exporting a single `exposeAPIInWorld` method.

The internal JS could then become
```ts
const contextBridge: Electron.ContextBridge = {
  exposeInMainWorld: (key: string, api: any) => {
    checkContextIsolationEnabled();
    return binding.exposeAPIInWorld(0, key, api);
  },
  exposeInIsolatedWorld: (worldId: number, key: string, api: any) => {
    checkContextIsolationEnabled();
    return binding.exposeAPIInWorld(worldId, key, api);
  }
};
```